models/example/hello2.sql

```diff
@@ -0,0 +1,6 @@
+{{ config(materialized='table',transient=true)}}
+
+select *
+from 
+SNOWFLAKE_SAMPLE_DATA.TPCH_SF1.ORDERS
+where O_TOTALPRICE>9000
```

Comment:
Try to use jinja functions instead of naming with literals . Explore How to use `source('source_name', 'table_name')`  and `schema.yml` file .

Reply:
@pradee29047 , can you work on this ? So we can close this part .

Reply:
I tried to do use jinja function and explored How to use source('source_name', 'table_name')  but got some error , will try again

Reply:
Done with using source jinja function {{ source(source_name, table_name) }} please have look.